R/app_template.R

```diff
@@ -115,6 +115,17 @@ shinyAppTemplate <- function(path = NULL, examples = "default", dryrun = FALSE)
     return(invisible())
   }
 
+  if ("shinytest" %in% examples) {
+    if (system.file(package = "shinytest") != "" &&
+        packageVersion("shinytest") <= "1.4.0")
+    {
+      message(
+        "The tests/shinytest directory needs shinytest 1.4.0 or later to work properly.\n",
+        "You currently have shinytest ", packageVersion("shinytest"), " installed."
+      )
```

Comment:
Should this be a `stop()`?

Reply:
Wondering if the second line should be displayed if the package is not installed.

Reply:
I don't think it should be a stop -- we shouldn't force people to change their system for this. But they should know about it.

I agree the second line shouldn't be displayed if shinytest is not installed.